libraries/chain/abi_serializer.cpp

```diff
@@ -207,6 +207,19 @@ namespace eosio { namespace chain {
       return ends_with(type, "[]");
    }
 
+   bool abi_serializer::is_szarray(const string_view& type)const {
```

Comment:
Good to have is_szarray() outside abieos in eos/libaries/chain/,   have you ever tried to verify nodoes can accept std::array<T,N>?  e.g have a test contract which has an action that uses std::array in its function body, and then use cleos command to publish that contract into nodos

Reply:
Good concern.  Actually the test case is in CDT develop branch, eosio.cdt/tests/unit/test_contracts/array_tests.cpp,  I did test some actions manually it works, hope it meet your concern.